### PR TITLE
Feature/documentation changes

### PR DESCRIPTION
**Pull request by muroc**

### CODE REVIEW

**Comment by wgromniak** [comment]
Cool.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,88 @@
...
-A simple example of how to use the API is available in 
-[Simple Trading Tutorial](docs/tutorials/simple_trading.py.md) - it shows how to set it up using
-[Autobahn](http://crossbar.io/autobahn/) WebSocket implementation on top of 
-[Twisted](https://www.twistedmatrix.com/). The tutorial is written in literate programming style and
-translates to [simple_trading.py](examples/simple_trading.py) script which is ready to run (and is
-actually run by the end-to-end tests).
+ * The API hides all technical concerns (transmission, encryption, serialization,
+   threading and connection) from the user so that all programming efforts may be
+   **concentrated on trading logic**.
+ * We provide classes for easy integration with [Twisted][twisted] which is one of the best
```
Comment (zarazek) [inline README.md]: Missing comma before 'which'

File: README.md [hunk 2]
```diff
@@ -1,40 +1,88 @@
...
+   frameworks for programming network communication in Python. Twisted uses asynchronuous
+   input/output which makes our API quite fast.
```
Comment (zarazek) [inline README.md]: ,which again

**Comment by wgromniak** [comment]
Let's merge this PR?

**Comment by wgromniak** [comment]
Merging, as discussed with @muroc.

**Pull request merged**